bundles/org.openhab.binding.sensibo/src/main/java/org/openhab/binding/sensibo/internal/client/RequestLogger.java

```diff
@@ -99,18 +99,16 @@ private void dump(final Request request, String[] stringsToRemove) {
                 }
 
                 String dataToLog = group.toString();
-                scambleAndLog(stringsToRemove, dataToLog);
+                scrambleAndLog(stringsToRemove, dataToLog);
             });
         }
     }
 
-    private void scambleAndLog(String[] stringsToRemove, String dataToLog) {
+    private void scrambleAndLog(String[] stringsToRemove, String dataToLog) {
         for (String stringToRemove : stringsToRemove) {
-            if (stringToRemove != null) {
-                dataToLog = dataToLog.replace(stringToRemove, "<HIDDEN>");
-            }
+            dataToLog = dataToLog.replace(stringToRemove, "<HIDDEN>");
```

Comment:
The compiler complains about storing a reference to a method argument. To fix this, you can store `dataLog` to a local variable at the beginning of this method.